### DISCUSSION TITLE
error with install

### DISCUSSION THREAD

**Discussion by plumbure** [OutlineFoundation/outline-sdk#284] [Q&A]
Docker installation failed, please visit https://docs.docker.com/install for instructions.
Sorry! Something went wrong. If you can't figure this out, please copy and paste all this output into the Outline Manager
screen, and send it to us, to see if we can help you.
how fix?

![image](https://github.com/user-attachments/assets/8a985b81-3db8-4f86-a24e-80f941d445cc)

**Comment by fortuna** [comment]
You need to install curl
